### ISSUE TITLE
problem with lonestarGPU

### ISSUE THREAD

**Issue by amrelhelw** [gpgpu-sim/gpgpu-sim_distribution#144]
I get the following error when i try to execute bfs in lonestarGPU.

./bfs: relocation error: ./bfs: symbol cudaCreateTextureObject version libcudart.so.9.1 not defined in file libcudart.so.9.1 with link time reference

Any help

**Comment by theverytak** [comment]
@mattsinc Is there any works done for patching the newer version of lonestargpu?

**Comment by mattsinc** [comment]
As far as I know there isn't a newer version of LonestarGPU, or at least the newer version doesn't require a different patch.  Have you found differently?

Either way, the first thing to do would be to try to apply the patch linked above and see if it resolves your issues (perhaps you've done this?).

Matt

**Comment by amrelhelw** [comment]
Good afternoon,

I apologize for my late response. I tried to apply the patch on lonestar for gpgpu-sim but unfortunately it did not work. I reached with lonestar development team and they informed me that they have updated lonestar benchmarks and the patch is no longer suitable.

I have tried several times to fix lonestar patch but I failed. I stopped using lonestar benchmarks as fixing the patch was taking a lot of time and it was not the main purpose of my current work. I switched to use other benchmarks such as polybench and GraphBig benchmarks.

There were some missing instructions implementation in “instructions.cc” files that must be added in order to make polybench work.

If you need more details about fixes and new implemented instructions, kindly feel free to contact me. I am currently spending winter break outside of USA and I have limited connectivity to the internet. I will be back to the United States after January 20, 2020.

If you are interested, I can discuss with you my design related to speeding up benchmarks simulation time.

Best regards,

Sent from my iPhone

On Jan 8, 2020, at 10:33 PM, Matt Sinclair <notifications@github.com> wrote:

﻿

As far as I know there isn't a newer version of LonestarGPU, or at least the newer version doesn't require a different patch. Have you found differently?

Either way, the first thing to do would be to try to apply the patch linked above and see if it resolves your issues (perhaps you've done this?).

Matt

**Comment by mattsinc** [comment]
Ok, good to know.  Is this a different version from the one already on Github: https://github.com/gpgpu-sim/gpgpu-sim_simulations/tree/master/benchmarks/src/cuda/lonestargpu-2.0?  If so, then you'll have to add the instructions that are missing, or open a ticket and hope someone else will.

Matt

**Comment by amrelhelw** [comment]
Yes, the new version is not on Github. Once it is published i will try to add the instructions and open a ticket
